### PR TITLE
Update monitoring explainer

### PR DESCRIPTION
**Pull request by akshaypundle**
- Added table of implemented metrics along with noising characteristics
 - Added code references for metrics definition, deployment and configiguration
 - Added section for GCP config including figures for sample GCP dashboard
 - Updated authors

### CODE REVIEW

File: monitoring_protected_audience_api_services.md
Comment (xinggao01) [inline monitoring_protected_audience_api_services.md:1]: "Common attributes" table need to be updated.

Comment (akshaypundle) [reply monitoring_protected_audience_api_services.md:1]: updated.

**Comment by xinggao01** [comment]
LGTM

**Review by xinggao01 — APPROVED** [review]

File: monitoring_protected_audience_api_services.md [hunk 1]
```diff
@@ -11,59 +15,164 @@ privacy guarantees of such a system, traditional ways of monitoring are not
...
+
+These are metrics tracked on the Seller front end B&A servers.
+
+| Metric                                           | Description                                                                      | Instrument    | Noising        | Attributes      |
+|--------------------------------------------------|----------------------------------------------------------------------------------|---------------|----------------|-----------------|
+| initiated_request.auction.duration_ms            | Total duration request takes to get response back from Auction server            | Histogram     | Noised with DP |                 |
+| initiated_request.auction.errors_count_by_status | Initiated requests by auction that resulted in failure partitioned by Error Code | UpDownCounter | Noised with DP | Absl error code |
+| initiated_request.auction.size_bytes             | Size of the initiated Request to Auction server in Bytes                         | Histogram     | Noised with DP |                 |
+| initiated_request.bfe.errors_count_by_status     | Initiated requests by KV that resulted in failure partitioned by Error Code      | UpDownCounter | Noised with DP | Absl error code |
```
Comment (roopalna) [inline monitoring_protected_audience_api_services.md:110]: The description for this metric should be Initiated requests by "BFE" that resulted.....

File: monitoring_protected_audience_api_services.md [hunk 2]
```diff
@@ -11,59 +15,164 @@ privacy guarantees of such a system, traditional ways of monitoring are not
...
+
+### Buyer Frontend Metrics
+
+These are metrics tracked on the BFE B&A servers.
+
+| Metric                                           | Description                                                                 | Instrument    | Noising        | Attributes      |
+|--------------------------------------------------|-----------------------------------------------------------------------------|---------------|----------------|-----------------|
+| bfe.error_code                                   | Number of errors in the BFE server by error code                            | UpDownCounter | Noised with DP | Error code      |
+| initiated_request.bidding.duration_ms            | Total duration request takes to get response back from bidding server       | Histogram     | Noised with DP |                 |
+| initiated_request.bidding.errors_count_by_status | Initiated requests by KV that resulted in failure partitioned by Error Code | UpDownCounter | Noised with DP | Absl error code |
```
Comment (roopalna) [inline monitoring_protected_audience_api_services.md:133]: Can you change the description for this metric too - Initiated Request to "Bidding" that resulted in failure.....

Comment (akshaypundle) [reply monitoring_protected_audience_api_services.md]: Done - just to confirm, these are requests initiated "to" bidding, not by bidding, right?

File: monitoring_protected_audience_api_services.md (hunk 1 above)
Comment (akshaypundle) [reply monitoring_protected_audience_api_services.md]: done.

File: monitoring_protected_audience_api_services.md (hunk 2 above)
Comment (roopalna) [reply monitoring_protected_audience_api_services.md]: Yes

**Review by roopalna — APPROVED** [review]

**Pull request merged**